src/System.Private.ServiceModel/src/System/ServiceModel/Channels/RTHttpClientHandler.cs

```diff
@@ -0,0 +1,475 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
This isn't a partial class of HttpClientHandler but is a full implementation in its own right so those naming conventions don't really apply. Having HttpClientHandler.netcore50.cs implies there's a base file without the suffix.